_upgrade-to/snapshot-migrate.md

```diff
@@ -9,3 +9,9 @@ nav_order: 5
 One popular approach is to take a [snapshot]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore) of your Elasticsearch OSS 6.x or 7.x indexes, [create an OpenSearch cluster]({{site.url}}{{site.baseurl}}/opensearch/install/), restore the snapshot on the new cluster, and point your clients to the new host.
 
 The snapshot approach can mean running two clusters in parallel, but lets you validate that the OpenSearch cluster is working in a way that meets your needs prior to modifying the Elasticsearch OSS cluster.
+
+Elasticsearch OSS version | Snapshot migration path
+:--- | :--- 
+5.x | Upgrade to 5.6, upgrade to 6.8, reindex all 5.x indexes, make snapshot and restore in OpenSearch 1.x.
```

Comment:
Would the user have to upgrade from OpenSearch 1.x to 2.x in order to use 2.x?

Reply:
Yes, but this is covered by Upgrading OpenSearch section of the documentation